prow.sh

```diff
@@ -1066,15 +1066,21 @@ main () {
                 docker tag "$i:latest" "$i:csiprow" || die "tagging the locally built container image for $i failed"
             done
 
-            if [ -e deploy/kubernetes/rbac.yaml ]; then
+            if [ -e "$(pwd)/deploy/kubernetes/rbac.yaml" ] || [ -e "$(pwd)/deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml" ]; then
                 # This is one of those components which has its own RBAC rules (like external-provisioner).
                 # We are testing a locally built image and also want to test with the the current,
                 # potentially modified RBAC rules.
+
+                # snapshot-controller RBAC is handled by the cluster manager
```

Comment:
Can you add a link here that provides more details for how the cluster manager does that?

Reply:
In the case of a pull job, we want to test all local changes from the pr, so I think we still want to apply local rbac changes

Reply:
Yeah, it's mentioned in the new readme. 
https://github.com/kubernetes-csi/external-snapshotter/blob/master/README.md#overview
> The snapshot controller is deployed by the Kubernetes distributions and is responsible for watching the VolumeSnapshot CRD objects and manges the creation and deletion lifecycle of snapshots.

Here's where prow.sh installs the snapshot-controller and it's RBAC:
https://github.com/kubernetes-csi/csi-release-tools/blob/master/prow.sh#L727

This happens right after cluster setup, along with the necessary CRD.
https://github.com/kubernetes-csi/csi-release-tools/blob/master/prow.sh#L1089
https://github.com/kubernetes-csi/csi-release-tools/blob/master/prow.sh#L1146

Reply:
> In the case of a pull job, we want to test all local changes from the pr, so I think we still want to apply local rbac changes

Ah yeah, will change this to handle the snapshot-controller.

Reply:
discussed offline, snapshot-controller will be addressed separately. so this pr should just fix csi-snapshotter sidecar testing

Reply:
Issue for this: https://github.com/kubernetes-csi/csi-release-tools/issues/79